### PR TITLE
Moving notice for the stratum dependencies

### PR DESCRIPTION
**Pull request by ffoulkes**
Announce our intention to move the Stratum dependencies to their own repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ upstreaming.
 It moves the P4-specific components of the architecture to a separate
 process called `infrap4d`.
 
+## Upcoming changes
+
+The Stratum dependencies will be moving from the `setup` directory to their
```
Comment (nupuruttarwar) [inline README.md:17]: typo: their 2 times?

**Review by nupuruttarwar — APPROVED** [review]
minor comment, rest LGTM

File: README.md (hunk 1 above)
Comment (ffoulkes) [reply README.md]: Good catch. Fixed. Made a couple of clarifying edits at the same time.

**Review by 5abeel — APPROVED** [review]
LGTM

**Pull request merged**